R/run_example.R

```diff
@@ -1,457 +1,458 @@
+            ##################### Create second panel################################
+            shiny::tabPanel(
+              "Text Types",
+              value = "text_types",
+              gov_layout(
+                size = "two-thirds",
+                backlink_Input("back1"),
+                heading_text("Page 2", size = "l"),
+                label_hint("label2", "These are some examples of the types of user
                    text inputs that you can use"),
-            heading_text("date_Input", size = "s"),
-            date_Input(
-              inputId = "date1",
-              label = "What is your date of birth?",
-              hint_label = "For example, 31 3 1980"),
-            heading_text("text_Input", size = "s"),
-            text_Input(inputId = "txt1", label = "Event name"),
-            heading_text("text_area_Input", size = "s"),
-            text_area_Input(
-              inputId = "text_area1",
-              label = "Can you provide more detail?",
-              hint_label = "Do not include personal or financial information,
-          like your National Insurance number or credit card details."),
-            text_area_Input(
-              inputId = "text_area2",
-              label = "How are you today?",
-              hint_label = "Leave blank to trigger error",
-              error = T,
-              error_message = "Please do not leave blank",
-              word_limit = 300),
-            heading_text("button_Input", size = "s", id = "button_input_text_types"),
-            button_Input("btn2", "Go to next page"),
-            button_Input("btn3", "Check for errors", type = "warning")
-          )
-        ),
+                heading_text("date_Input", size = "s"),
+                date_Input(
+                  inputId = "date1",
+                  label = "What is your date of birth?",
+                  hint_label = "For example, 31 3 1980"
+                ),
+                heading_text("text_Input", size = "s"),
+                text_Input(inputId = "txt1", label = "Event name"),
+                heading_text("text_area_Input", size = "s"),
+                text_area_Input(
+                  inputId = "text_area1",
+                  label = "Can you provide more detail?",
+                  hint_label = "Do not include personal or financial information,
+          like your National Insurance number or credit card details."
+                ),
+                text_area_Input(
+                  inputId = "text_area2",
+                  label = "How are you today?",
+                  hint_label = "Leave blank to trigger error",
+                  error = T,
+                  error_message = "Please do not leave blank",
+                  word_limit = 300
+                ),
+                heading_text("button_Input", size = "s", id = "button_input_text_types"),
+                button_Input("btn2", "Go to next page"),
```

Comment:
go to next page doesn't do anything here (also replicates the button we have on the previous page - could remove?)

Reply:
Yeah, just removed as I don't think we need, we have the other example of it